ipn/ipnlocal/local.go

```diff
@@ -1962,7 +1968,7 @@ func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWa
 	b.mu.Lock()
 	const initialBits = ipn.NotifyInitialState | ipn.NotifyInitialPrefs | ipn.NotifyInitialNetMap
 	if mask&initialBits != 0 {
-		ini = &ipn.Notify{Version: version.Long()}
+		ini = &ipn.Notify{Version: version.Long(), SessionID: sessionID}
```

Comment:
I think this'll break swift unless you make the session ID opt-in with a new bitmask flag.

also, you'll want to register this somewhere on LocalBackend and call LocalBackend when this Watch call dies.

Reply:
New fields as I see in this PR are fine, Swift will simply ignore them. The problem is when a field is non-optional in Swift, and the server stops sending it: that will cause a failure to decode.